backend/config.py

```diff
@@ -1,29 +1,31 @@
 import os
 
 class Config:
-    APP_NAME = "Move Up CRM Tool"
+    APP_NAME = "Move Up Sign-Up Tool"
```

Comment:
Maybe add SMS in here somewhere, e.g. `Move Up SMS Sign-Up Tool`

Reply:
Resolved in the new commit.